### PR TITLE
Document newly supported authentication schemes

### PR DESCRIPTION
**Pull request by fbiville**
Note: do not merge before https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1068

### CODE REVIEW

File: common-content-dataflow/modules/ROOT/partials/prerequisites.adoc [hunk 1]
```diff
@@ -11,19 +11,80 @@ If you don't have an instance yet, you have two options:
 - sign-up for a free link:https://neo4j.com/cloud/aura-free/[AuraDB] instance
 - install and self-host Neo4j in a location that is publicly accessible (see link:https://neo4j.com/docs/operations-manual/current/installation/[Neo4j -> Installation]) with port 7687 open (Bolt protocol)
 
-Either way, you then need to create a file containing your database connection information in JSON format (basic authentication is the only supported method).
+Either way, you then need to create a file containing your database connection information in JSON format.
 This file will go into your Cloud Storage bucket.
 
-.neo4j-connection-info.json
-[source, JSON]
+[.tabbed-example]
```
Comment (fbiville) [inline common-content-dataflow/modules/ROOT/partials/prerequisites.adoc:17]: I cannot make the display of tabs work at all.
Would you know what I am missing @stefano-ottolenghi?

Comment (stefano-ottolenghi) [reply common-content-dataflow/modules/ROOT/partials/prerequisites.adoc:17]: Apparently the syntax for each tab is `[.include-with-textWithDashes]`, where `textWithDashes` is a `[^\s]*` string in which dashes get replaced by spaces. See https://github.com/neo4j-documentation/docs-ui/blob/master/src/js/08-tabs-block.js

**Review by stefano-ottolenghi — APPROVED** [review]
I've fixed the tabbed example and added a brief description for each tab with pointers to more info.

File: common-content-dataflow/modules/ROOT/partials/prerequisites.adoc (hunk 1 above)
Comment (fbiville) [reply common-content-dataflow/modules/ROOT/partials/prerequisites.adoc:17]: oh ok, thanks for looking into this!

**Pull request merged**